src/universal/modules/admin/containers/Auth/AuthLoginAs.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {cashay} from 'cashay';
+import Helmet from 'react-helmet';
+import ActionHTTPTransport from 'universal/utils/ActionHTTPTransport';
+import LoadingView from 'universal/components/LoadingView/LoadingView';
+import {getAuthQueryString, getAuthedOptions} from 'universal/redux/getAuthedUser';
+
+async function createImposterToken(code) {
+  cashay.create({httpTransport: new ActionHTTPTransport()});
+  const options = {variables: {code}};
+  await cashay.mutate('createImposterToken', options);
```

Comment:
@mattkrick can you once again help me with the steps to avoid the exception I'm seeing on the server:

```
DEBUG GraphQL Error: [ { GraphQLError: Field "createImposterToken" of type "User" must have a selection of subfields. Did you mean "createImposterToken { ... }"?
```

Shouldn't this customMutation and mutationHandler be set up to fetch `id` and `jwt` from the server?